recipes/ctat-lncrna/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# ctat_lncrna_DIR_NAME="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+# The above is an error. The following is correct.
+ctat_lncrna_DIR_NAME="$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+ctat_lncrna_INSTALL_PATH="$PREFIX/share/$ctat_lncrna_DIR_NAME"
+# Make the install directory and move the ctat_lncrna files to that location.
+mkdir -p $PREFIX/bin
+mkdir -p $ctat_lncrna_INSTALL_PATH
+#copy to INSTALL_PATH
+cp -R $SRC_DIR/LICENSE.txt $SRC_DIR/lncrna $SRC_DIR/SciEDPipeR $ctat_lncrna_INSTALL_PATH
```

Comment:
why do you copy the license as well? Conda will do this for you if you have specified it in the meta.yaml, which you have done :)
Why not copying everything into PREFIX/bin directly?

Reply:
The reason for not copying into bin directly is as follows:
Most of the CTAT tools use different versions of SciEDPipeR, so if a user installs 2 of CTAT tools then version conflicts may occur hence my attempt at keeping everything in install path and invoking from there.